Scripts/WikiUpdater.cs

```diff
@@ -0,0 +1,415 @@
+{
+    private const string ORGANELLE_CATEGORY = "https://wiki.revolutionarygamesstudio.com/wiki/Category:Organelles";
+    private const string WIKI_FILE = "simulation_parameters/common/wiki.json";
+    private const string ENGLISH_TRANSLATION_FILE = "locale/en.po";
+    private const string TEMP_TRANSLATION_FILE = "en.po.temp_wiki";
+
+    /// <summary>
+    ///   Compounds to replace with custom BBCode when appearing in bold on wiki pages.
+    /// </summary>
+    private static readonly string[] CustomBbcodeCompounds =
+    {
+        // TODO: get these values from English translations of the names in organelles.json
+        "ATP",
+        "Ammonia",
+        "Carbon Dioxide",
+        "Glucose",
+        "Hydrogen Sulfide",
+        "Iron",
+        "Mucilage",
+        "Nitrogen",
+        "Oxygen",
+        "OxyToxy",
+        "Phosphates",
+        "Sunlight",
+        "Temperature",
+    };
+
+    /// <summary>
+    ///   Inserts selected content from the online wiki into the game files. See
+    ///   https://wiki.revolutionarygamesstudio.com/wiki/Thriveopedia for instructions.
+    /// </summary>
+    public static async Task<bool> Run(CancellationToken cancellationToken)
+    {
+        var organellesRootTask = FetchOrganellesRootPage(cancellationToken);
+        var organellesTask = FetchOrganellePages(cancellationToken);
+
+        var organellesRoot = await organellesRootTask;
+        var organelles = await organellesTask;
+
+        var untranslatedWiki = new Wiki(
+            organellesRoot.UntranslatedPage,
+            organelles.Select(o => o.UntranslatedPage).ToList());
+
+        await JsonWriteHelper.WriteJsonWithBom(WIKI_FILE, untranslatedWiki, cancellationToken);
+        ColourConsole.WriteSuccessLine($"Updated wiki at {WIKI_FILE}, running translations update");
+
+        var localizationUpdater = new LocalizationUpdate(new LocalizationOptionsBase { Quiet = true });
+        if (!await localizationUpdater.Run(cancellationToken))
+            return false;
+
+        ColourConsole.WriteSuccessLine("Translations update succeeded, inserting English strings for wiki content");
+
+        await InsertTranslatedPageContent(
+            organelles.Append(organellesRoot).ToList(),
+            cancellationToken);
+
+        ColourConsole.WriteSuccessLine("Successfully updated English translations for wiki content");
+
+        return true;
+    }
+
+    private static async Task<TranslationPair> FetchOrganellesRootPage(CancellationToken cancellationToken)
+    {
+        ColourConsole.WriteInfoLine("Fetching organelles root page");
+
+        var body = (await HtmlReader.RetrieveHtmlDocument(ORGANELLE_CATEGORY, cancellationToken)).Body!;
+
+        var sections = GetMainBodySections(body);
+        var untranslatedSections = sections.Select(section => UntranslateSection(section, "ORGANELLES_ROOT")).ToList();
+
+        var untranslatedPage = new Wiki.Page(
+            "WIKI_PAGE_ORGANELLES_ROOT",
```

Comment:
Usually in Thrive code we use the style where as many parameters are packed on a single line instead of this "chop" style (as called by Jetbrains). So this isn't exactly consistent coding style with the rest of the codebase, but as this is in the scripts this is a bit less serious of a style difference in my opinion. Up to you if you want to fix this.